doc/sphinx/source/project/introduction.rst

```diff
@@ -4,189 +4,97 @@
-applied.  The result is a more efficient execution that does not require any 
-manual kernel integration work for most hardware targets. 
-
-What follows here is more detail about how our solution addresses these 
-problems. 
+The nGraph Compiler stack is designed to address the first two problems. nGraph applies graph-level optimizations by taking the computational graph from a deep learning framework such as TensorFlow\* and reconstructs it with nGraph 
+:abbr:`Intermediate Representation (IR)`. nGraph IR centralizes computational graphs from various frameworks and provides a unified way to connect backends for targeted hardware. To address the third problem, nGraph is integrated with PlaidML, *a lorem ipsum*, to generate code in LLVM, OpenCL, OpenGL, Cuda, and Metal. Low-level optimizations are automatically applied to the generated code, resulting in a more efficient execution that does not require manual kernel integration for most hardware targets. 
 
+The following three sections explore the main problems of kernel libraries in more detail and describe how nGraph addresses them.
 
-Problem: Absence of graph-level optimizations
+Problem 1: Absence of graph-level optimizations
 ---------------------------------------------
 
-The diagram below illustrates a simple example of how a deep learning 
-framework, when integrated with a kernel library, is capable of running each 
-operation in a computational graph optimally, but the graph itself may not be 
-optimal: 
+The example diagram below shows how a deep learning framework, when integrated with a kernel library, can optimally run each operation in a computational graph, but the graph itself may not be optimal.
 
 .. _figure-A:
 
 .. figure:: ../graphics/intro_graph_optimization.png
    :width: 555px
    :alt: 
 
-The computation is constructed to execute ``(A+B)*C``, but in the context of 
-nGraph, we can further optimize the graph to be represented as ``A*C``. From the 
-first graph shown on the left, the operation on the constant ``B`` can be 
-computed at the compile time (known as constant folding), and the graph can be 
-further simplified to the one on the right because the constant has value of 
-zero. Without such graph-level optimizations, a deep learning framework with a 
-kernel library will compute all operations, and the resulting execution will be 
-suboptimal. 
+The computation is constructed to execute ``(A+B)*C``, but in the context of nGraph, we can further optimize the graph to be represented as ``A*C``. From the first graph shown on the left, the operation on the constant ``B`` can be computed at the compile time (known as constant folding), and the graph can be further simplified to the one on the right because the constant has a value of zero. Without such graph-level optimizations, a deep learning framework with a kernel library will compute all operations, resulting in suboptimal execution. 
```

Comment:
"because the constant has a value of zero *(known as algebraic simplification)*"